river/Output.zig

```diff
@@ -70,7 +71,7 @@ pub const PendingState = struct {
         var count: u32 = 0;
         while (it.next()) |other| {
             if (other == focused_view) break;
-            count += 1;
+            if (other.current.tags & self.tags > 0) count += 1;
```

Comment:
Then needs to check `pending` tags not current tags as we are working with pending output state.